pandas/tests/window/moments/conftest.py

```diff
@@ -18,3 +18,51 @@ def binary_ew_data():
 @pytest.fixture(params=[0, 1, 2])
 def min_periods(request):
     return request.param
+
+
+base_functions_list = [
```

Comment:
can you document the tuples here (e.g. what they refer to)

Reply:
yep, i added in the docs